wp-content/themes/flightfindertheme/db_connect.php

```diff
@@ -0,0 +1,4 @@
+<?php
```

Comment:
You don't need to connect to the db by yourself, this is done my wordpress. Use `$wpdb`